contracts/src/v0.8/llo-feeds/dev/FeeManager.sol

```diff
@@ -200,8 +208,10 @@ contract FeeManager is IFeeManager, ConfirmedOwner, TypeAndVersionInterface {
 
         //check that the contract has enough LINK before paying the fee
         if (reward.amount > IERC20(i_linkAddress).balanceOf(address(this))) {
-          // If not enough LINK on this contract to forward for rewards, fire this event and
-          // call onFeePaid out-of-band to pay out rewards
+          // If not enough LINK on this contract to forward for rewards, tally the deficit to be paid by out-of-band LINK
+          unchecked {
```

Comment:
Do we really need unchecked here?

Reply:
Primary intention is gas savings since it's an easy change.